docker-compose-services/php8/README.md

```diff
@@ -0,0 +1,15 @@
+# PHP8 (while in alpha/beta/etc)
+
+DDEV-Local will directly support PHP8 as soon as possible, but not likely until the full release in late 2020. At that time this recipe will be obsolete.
+
+In the meantime, you can use the [official docker php image](https://hub.docker.com/_/php) (currently php:8.0.0alpha3-fpm-buster) to serve PHP.  This can be updated as new versions come out.
+
+1. Copy [docker-compose.php8.yaml](docker-compose.php8.yaml) to your project's .ddev folder.
+2. Copy [nginx-site.conf](nginx-site.conf) to your .ddev/nginx_full directory (overwriting the generated file there)
+3. `ddev start`
+
+Note that this introduces a container named `php`, which you can accesss via `ddev ssh -s php`. It has the exact same code mounted in the same place as in the `web` container, at `/var/www/html`.
```

Comment:
```suggestion
Note that this introduces a container named `php`, which you can accesss via `ddev ssh -s php`. It has the exact same code mounted in the same place as in the `web` container, at `/var/www/html`. It is just the vanilla upstream docker container, so things like composer, etc. all have to be manually installed.
```

Reply:
I wouldn't actually recommend to people to go into the php container though, so there's that. Why wouldn't you just use composer as always? The same code is mounted into both containers, can be changed on the host, in web container, or in php container. It's just being *interpreted* in the php container.